sources/ConflictWindow.cpp

```diff
@@ -19,7 +19,7 @@ static const unsigned kReplace = 'RPLC';
 static const unsigned kSkip = 'SKIP';
 
 ConflictWindow::ConflictWindow(const char* file)
-	: BWindow(BRect(0, 0, 0, 0), "", B_BORDERED_WINDOW, B_WILL_ACCEPT_FIRST_CLICK | B_AUTO_UPDATE_SIZE_LIMITS)
+	: BWindow(BRect(0, 0, 0, 0), "", B_MODAL_WINDOW, B_WILL_ACCEPT_FIRST_CLICK | B_AUTO_UPDATE_SIZE_LIMITS)
```

Comment:
Yeah, I missed the the tiny breaks in the corner. The B_NOT_RESIZABLE is back in. Please let me know how the new feature (issue #17) works. Thanks!